tools/esp8266/hmSystem.h

```diff
@@ -19,7 +19,8 @@ class HmSystem {
         RadioType Radio;
         typedef BUFFER BufferType;
         BufferType BufCtrl;
-        InfoCmdType InfoCmd;
+        InfoCmdType NextInfoCmd; // For cmd-queue FIFO with one place
```

Comment:
Nextinfocmd ist doch der punkt an dem die API queue in den normalen prozess mündet